Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -73,6 +80,36 @@ class SectionEditorViewController: UIViewController {
         inputViewsController.keyboardDidHide()
     }
     
+    private func setupFocusNavigationView() {
+        #warning("todo: localize")
+        
+        focusNavigationView.isHidden = true
+        focusNavigationView.configure(text: "Find and replace", traitCollection: traitCollection)
+        focusNavigationView.delegate = self
+        focusNavigationView.apply(theme: theme)
+        
+        focusNavigationView.translatesAutoresizingMaskIntoConstraints = false
+        navigationController?.navigationBar.addSubview(focusNavigationView)
```

Comment:
This seems gross to me and I'm not sure if it'll be accessible yet. I'm open to other ideas on it. In terms of responsibility this code might make more sense living in WMFThemeableNavigationController?

Reply:
Could this tell the navigation controller to hide the navigation bar and then display the focus navigation view at the top of this view controller's view? That might be an easier way to address the responsibility issue

Reply:
Alternatively, this could become a subclass of `ViewController` and utilize the included `NavigationBar` - `focusNavigationView` could be the `underBarView` and it has the ability to hide/show individual components with `setNavigationBarPercentHidden(_ navigationBarPercentHidden: CGFloat, underBarViewPercentHidden: CGFloat, extendedViewPercentHidden: CGFloat, topSpacingPercentHidden: CGFloat...`

Reply:
Either way, not necessary for this PR, can be a follow-on

Reply:
@joewalsh I went with your first suggestion (in https://github.com/wikimedia/wikipedia-ios/pull/2987). The ViewController superclass would move some of the theming and keyboard handling events around, which seems a little risky at this point.